functests/1_performance/cpu_management.go

```diff
@@ -59,7 +61,7 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 
 		Expect(profile.Spec.CPU.Reserved).NotTo(BeNil())
 		reservedCPU = string(*profile.Spec.CPU.Reserved)
-		reservedCPUSet, err := cpuset.Parse(reservedCPU)
+		reservedCPUSet, err = cpuset.Parse(reservedCPU)
```

Comment:
Good catch! This sees like is not related to the patch, I would move it to a different patch as a bugfix, same pr though.

Reply:
:) I use this variable in new test, so not sure it's good idea to make several PRs for it

Reply:
I means several patches in the same PR, when looking at git logs will be easier to track bugs

Reply:
I created 2 commits: 
1st - is all about fixes for existing CPU affinity test (changed pid, removed colon and changed test_id)
2nd - adding new test